lm_eval/api/metrics.py

```diff
@@ -243,6 +243,15 @@ def perplexity_fn(items):  # This is a passthrough function
     return items
 
 
+@register_metric(
+    metric="perplexity_rolling",
```

Comment:
```suggestion
    metric="token_perplexity",
```

I'd love if we could call this metric `token_perplexity`, we should make it clear that this is token-level PPL! (I don't want people to simply decide to use `perplexity_rolling` thinking it's the "default", as any pick is a deliberate choice)